test/CMakeLists.txt

```diff
@@ -32,7 +33,7 @@ sleip_add_test(operations)
 sleip_add_test(fancy_pointer)
 target_link_libraries(fancy_pointer PRIVATE Threads::Threads)
 if (UNIX AND NOT APPLE)
-  target_link_libraries(fancy_pointer PRIVATE rt)
+  target_link_libraries(fancy_pointer PRIVATE Threads::Threads rt)
```

Comment:
Does "apple" not need `Threads::Threads`?

Reply:
This was a weird change. I re-compiled using clang w/o sanitizers and I got an error about missing DSO symbols. I'm not sure how CMake does it link flags but it seems like just linking against librt messes things up.

In the ideal case, this is only a Linux problem. I think OS X has its own thing whereas Linux requires linking against librt